package.json

```diff
@@ -0,0 +1,88 @@
+{
+	"name": "lana",
+	"displayName": "Apex Log Analyzer",
+	"version": "1.3.5",
+	"description": "Analyzer for Salesforce debug logs",
+	"keywords": [
+		"salesforce",
+		"apex",
+		"apexlog",
+		"debuglog"
+	],
+	"main": "out/Main.js",
+	"icon": "logo.png",
+	"publisher": "FinancialForce",
+	"author": "",
+	"license": "SEE LICENSE IN LICENSE.txt",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/financialforcedev/debug-log-analyzer.git"
+	},
+	"engines": {
+		"vscode": "^1.54.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"activationEvents": [
+		"onCommand:lana.loadLogFile",
+		"onCommand:lana.showLogFile"
+	],
+	"contributes": {
+		"commands": [
+			{
+				"command": "lana.loadLogFile",
+				"title": "Log: Load Apex Log For Analysis"
+			},
+			{
+				"command": "lana.showLogFile",
+				"title": "Log: Show Log Analysis"
+			}
+		],
+		"menus": {
+			"commandPalette": [
+				{
+					"command": "lana.showLogFile",
+					"when": "resourceExtname == '.log'"
+				}
+			],
+			"editor/context": [
+				{
+					"command": "lana.showLogFile",
+					"when": "resourceExtname == '.log'"
+				}
+			],
+			"explorer/context": [
+				{
+					"command": "lana.showLogFile",
+					"when": "resourceExtname == '.log'"
+				}
+			]
+		}
+	},
+	"scripts": {
+		"vscode:prepublish": "npm run compile",
+		"compile": "npm run log-viewer && tsc -p ./",
+		"watch": "tsc -watch -p ./",
+		"pretest": "npm run compile && npm run lint",
+		"lint": "eslint src --ext ts",
+		"test": "node ./out/test/runTest.js",
+		"log-viewer": "(cd log-viewer; npm ci && npm run build)"
+	},
+	"dependencies": {
+		"pkgforce": "1.3.0"
+	},
+	"devDependencies": {
+		"@types/vscode": "^1.54.0",
+		"@types/glob": "^7.1.3",
+		"@types/mocha": "^8.0.4",
+		"@types/node": "^12.11.7",
+		"eslint": "^7.19.0",
+		"@typescript-eslint/eslint-plugin": "^4.14.1",
+		"@typescript-eslint/parser": "^4.14.1",
+		"glob": "^7.1.6",
```

Comment:
Have you considered [fast-glob](https://github.com/mrmlnc/fast-glob) or [fdir](https://github.com/thecodrr/fdir) instead? Both are typescript and I have used fast-glob before and it is a lot faster than glob.

Reply:
Think this will have come from the yo generator which I used to setup, I didn't review these though so I will take a look at why it's there.

Reply:
Looks like it's only used in the test suite.

Reply:
I have removed since the there are no tests on the extension itself, just some for the log-viewer code.